sdk/aqueduct/aqueduct_client.py

```diff
@@ -417,12 +418,21 @@ def trigger(
         flow_id = parse_user_supplied_id(flow_id)
         self._api_client.refresh_workflow(flow_id, serialized_params)
 
-    def delete_flow(self, flow_id: Union[str, uuid.UUID]) -> None:
+    def delete_flow(
+        self,
+        flow_id: Union[str, uuid.UUID],
+        writes_to_delete: Optional[DefaultDict[uuid.UUID, List[WrittenObject]]] = None,
+        force: bool = False,
+    ) -> Dict[uuid.UUID, List[WritesDelete]]:
```

Comment:
Should we return any objects here? I don't think if the user would actually programmatically use this returned value  esp. that it's also an `uuid->list` mapping. If it's not intended for user to use the returned value, we should return None here (and it's also generally natural that a `delete` returns `None`)

It should be sufficient that we print out any deletion failures for saved object. We may actually want to raise exceptions in these failures.

Reply:
Returning None now and raising an exception if any table failed to delete